tests/unit/Promise.ts

```diff
@@ -310,7 +310,7 @@ export function addPromiseTests(suite: any, Promise: PromiseType) {
 				let dfd = this.async();
 				let error = new Error('foo');
 				let foreign = <any> {
-					then: function (f: Function) {
+					then: (f: Function) => {
```

Comment:
Attempting to fix an error, didn't work.
Test not returning `Error: Foo`, instead `ReferenceError: this is not defined`